src/lib/components/Stepper.react.js

```diff
@@ -0,0 +1,153 @@
+                    ["completedIcon", "description", "icon", "label", "progressIcon",]
+                );
+
+                console.log(renderedProps, index)
+                if (childType === "StepperStep") {
+                    return (
+                        <MantineStepper.Step
+                            {...omit(["children", "completedIcon",
+                                "description", "icon", "label",
+                                "progressIcon"], childProps)}
+                            {...renderedProps}
+                            key={index}
+                        >
+                            {childProps.children}
+                        </MantineStepper.Step>
+                    );
+                } else if (childType === "StepperCompleted") {
+                    return (
+                        <MantineStepper.Completed>
+                            {child}
+                        </MantineStepper.Completed>
+                    );
+                }
+            })}
+        </MantineStepper>
+    )
+
+}
+
+Stepper.displayName = "Stepper";
+
+Stepper.defaultProps = {};
+
+Stepper.propTypes = {
+    /**
+    * Active step index
+    */
+    active: PropTypes.number,
+    /**
+    * Breakpoint at which orientation will change from horizontal to vertical
+    */
+    breakpoint: PropTypes.oneOfType([
+        PropTypes.oneOf(["xs", "sm", "md", "lg", "xl"]),
+        PropTypes.number,
+    ]),
+    /**
+    * <Stepper.Step /> components only
+    */
+    children: PropTypes.node,
+    /**
+    * Often used with CSS to style elements with common properties
+    */
+    class_name: PropTypes.string,
+
+    /**
+    * Active and progress Step colors from theme.colors
+    */
+    color: PropTypes.oneOf([
+        "dark",
+        "gray",
+        "red",
+        "pink",
+        "grape",
+        "violet",
+        "indigo",
+        "blue",
+        "cyan",
+        "teal",
+        "green",
+        "lime",
+        "yellow",
+        "orange",
+    ]),
+    /**
+    * Step icon displayed when step is completed
+    */
+    completedIcon: PropTypes.node,
+    /**
+    * Content padding-top from theme.spacing or number to set value in px
+    */
+    contentPadding: PropTypes.oneOfType([
+        PropTypes.oneOf(["xs", "sm", "md", "lg", "xl"]),
+        PropTypes.number,
+    ]),
+    /**
+    * 	Icon position relative to step body
+    */
+    iconPosition: PropTypes.oneOf(["left", "right"]),
+    /**
+    * Step icon size in px
+    */
+    iconSize: PropTypes.number,
+    /**
+    * Called when step is clicked
+    */
+    // onStepClick: 
+
+    /**
+    * The ID of this component, used to identify dash components in callbacks
+    */
+    id: PropTypes.string,
+    /**
+    * Component orientation
+    */
+    orientation: PropTypes.oneOf(["horizontal", "vertifical"]),
+    /**
+    * 	Step icon displayed when step is in progress
+    */
+    progressIcon: PropTypes.node,
```

Comment:
For all props that you are going to use with renderDashComponents, use propType as PropTypes.any
```suggestion
    progressIcon: PropTypes.any,
```